src/systems/si/include/units/isq/si/prefixes.h

```diff
@@ -27,26 +27,26 @@
-struct zepto  : prefix<zepto, "z",             ratio(1, 1, -21)> {};
-struct atto   : prefix<atto,  "a",             ratio(1, 1, -18)> {};
-struct femto  : prefix<femto, "f",             ratio(1, 1, -15)> {};
-struct pico   : prefix<pico,  "p",             ratio(1, 1, -12)> {};
-struct nano   : prefix<nano,  "n",             ratio(1, 1,  -9)> {};
-struct micro  : prefix<micro, basic_symbol_text{"\u00b5", "u"}, ratio(1, 1,  -6)> {};
-struct milli  : prefix<milli, "m",             ratio(1, 1,  -3)> {};
-struct centi  : prefix<centi, "c",             ratio(1, 1,  -2)> {};
-struct deci   : prefix<deci,  "d",             ratio(1, 1,  -1)> {};
-struct deca   : prefix<deca,  "da",            ratio(1, 1,   1)> {};
-struct hecto  : prefix<hecto, "h",             ratio(1, 1,   2)> {};
-struct kilo   : prefix<kilo,  "k",             ratio(1, 1,   3)> {};
-struct mega   : prefix<mega,  "M",             ratio(1, 1,   6)> {};
-struct giga   : prefix<giga,  "G",             ratio(1, 1,   9)> {};
-struct tera   : prefix<tera,  "T",             ratio(1, 1,  12)> {};
-struct peta   : prefix<peta,  "P",             ratio(1, 1,  15)> {};
-struct exa    : prefix<exa,   "E",             ratio(1, 1,  18)> {};
-struct zetta  : prefix<zetta, "Z",             ratio(1, 1,  21)> {};
-struct yotta  : prefix<yotta, "Y",             ratio(1, 1,  24)> {};
+struct yocto  : prefix<yocto, "y",             pow<-24>(as_magnitude<10>())> {};
+struct zepto  : prefix<zepto, "z",             pow<-21>(as_magnitude<10>())> {};
+struct atto   : prefix<atto,  "a",             pow<-18>(as_magnitude<10>())> {};
+struct femto  : prefix<femto, "f",             pow<-15>(as_magnitude<10>())> {};
+struct pico   : prefix<pico,  "p",             pow<-12>(as_magnitude<10>())> {};
+struct nano   : prefix<nano,  "n",             pow<-9>(as_magnitude<10>())> {};
+struct micro  : prefix<micro, basic_symbol_text{"\u00b5", "u"}, pow<-6>(as_magnitude<10>())> {};
+struct milli  : prefix<milli, "m",             pow<-3>(as_magnitude<10>())> {};
+struct centi  : prefix<centi, "c",             pow<-2>(as_magnitude<10>())> {};
+struct deci   : prefix<deci,  "d",             pow<-1>(as_magnitude<10>())> {};
+struct deca   : prefix<deca,  "da",            pow<1>(as_magnitude<10>())> {};
+struct hecto  : prefix<hecto, "h",             pow<2>(as_magnitude<10>())> {};
+struct kilo   : prefix<kilo,  "k",             pow<3>(as_magnitude<10>())> {};
+struct mega   : prefix<mega,  "M",             pow<6>(as_magnitude<10>())> {};
+struct giga   : prefix<giga,  "G",             pow<9>(as_magnitude<10>())> {};
+struct tera   : prefix<tera,  "T",             pow<12>(as_magnitude<10>())> {};
+struct peta   : prefix<peta,  "P",             pow<15>(as_magnitude<10>())> {};
+struct exa    : prefix<exa,   "E",             pow<18>(as_magnitude<10>())> {};
+struct zetta  : prefix<zetta, "Z",             pow<21>(as_magnitude<10>())> {};
+struct yotta  : prefix<yotta, "Y",             pow<24>(as_magnitude<10>())> {};
```

Comment:
To be updated in the docs: https://mpusz.github.io/units/framework/units.html#prefixed-unit